code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -109,7 +109,7 @@
 /datum/spellbook_entry/rod_form
 	name = "Rod Form"
 	spell_type = /obj/effect/proc_holder/spell/targeted/rod_form
-	cost = 3
+	cost = 40
```

Comment:
This looks like an 'ided pls nerf' PR. The only tweak to wizard spellcosts is Rodform, which raises cost from 3 to 40.